README.md

```diff
@@ -81,6 +82,15 @@ With this:
 "hello world".length
 ```
 
+
+## Date
+Easily break down a date into components with our helper method:
+
+```swift
+let date = Date()
+date.components([.hour, .day. year. calendar])
```

Comment:
Calendar should probably be outside the array (or omitted in this bit of documentation)

Reply:
Missing some commas, misplaced some periods. Removing the component anyway for clarity. 👍